Commands/Responses.py

```diff
@@ -478,8 +478,7 @@ def toggle(self, chatMessage):
         @type chatMessage: IRCMessage
         """
         self.enabled = not self.enabled
-        return self.chat("Response '%s' %s" % (self.name, {True: "Enabled", False: "Disabled"}[self.enabled]),
-                         chatMessage)
+        return self.chat("Response \'{}\' {}".format(self.name, 'enabled' if self.enabled else 'disabled'), chatMessage)
```

Comment:
this would be better as `{!r}`, not `\'{}\'`. It would handle quotes, etc properly.